go.mod

```diff
@@ -76,7 +76,7 @@ require (
 	github.com/cenkalti/backoff/v3 v3.0.0 // indirect
 	github.com/cespare/xxhash/v2 v2.1.2 // indirect
 	github.com/chrismellard/docker-credential-acr-env v0.0.0-20220327082430-c57b701bfc08 // indirect
-	github.com/containerd/containerd v1.6.12 // indirect
+	github.com/containerd/containerd v1.6.18 // indirect; indirect // indirect
```

Comment:
@phisco any idea on why these extra indirect statements continue to be added? I’m thinking of just bumping manually for now.

Reply:
Checking 🤔